blurring_as_a_service/inference_pipeline/components/detect_and_blur_sensitive_data.py

```diff
@@ -19,6 +19,7 @@
 from blurring_as_a_service.settings.settings_helper import (  # noqa: E402
     setup_azure_logging,
 )
+from blurring_as_a_service.utils.lock_file import LockFile  # noqa: E402
```

Comment:
this import should come later, read comments on lines 30-31

Reply:
You are the master of logs for a reason 😄